File: portafolio-frontend/src/app/models/TblPerfilDTO.ts
import { Byte } from '@angular/compiler/src/util';
export class TblPerfilDTO {

  // CAMPO IDENTIFICADOR DE PERFIL
  idPerfil: number;

  // CAMPO IDENTIFICADOR DE UBIGEO NACIMIENTO DEL PERFIL
  idUbigeo: number;

  // CAMPO NUMERO DE DNI DEL PERFIL
  nuDniPerfil: number;

  // CAMPO NOMBRE DE PERFIL
  noPerfil: string;

  // CAMPO APELLIDOS DEL PERFIL
  apPerfil: string;

  // CAMPO DIRECCIÓN DEL PERFIL
  dirPerfil: string;

  // CAMPO TELEFONO DEL PERFIL
  telPerfil: number;

  // CAMPO CORREO ELECTRONICO DEL PERFIL
  emailPerfil: string;

  // CAMPO FECHA DE NACIMIENTO DEL PERFIL
  feNacimientoPerfil: Date;

  // NOMBRE ORIGINAL DE LA IMAGEN
  imgPerfil: string;

  // TIPO DE IMAGEN
  tipoImg: string;

  // NOMBRE DE LA IMAGEN
  codImg: Byte[];

  // DESCRIBIR SOBRE MÍ DE COMO SOY
  sobreMi: string;

  // RESUMEN DE LO QUE HAGO
  resumen: string;

  // DESCRIBIR TU EDUCACIÓN
  deEducacion: string;

  // DESCRIBIR TU EXPERIENCIA
  deExperiencia: string;

  // DESCRIBIR TU FORMACION
  deFormacion: string;

  // DESCRIBIR LOS IDIOMAS QUE USAS
  deIdioma: string;

  // DESCRIBIR TUS HABILIDADES
  deHabilidad: string;

  // DESCRIBIR TUS DATOS
  deDato: string;

  // CAMPO ESTADO DEL REGISTRO. LOS POSIBLES VALORES SON: "1" = ACTIVO Y "0" =
  // INACTIVO
  esRegistro: string;

  // CAMPO USUARIO CREADOR
  usCreacion: string;

  // CAMPO TERMINAL DE CREACIÓN
  ipCreacion: string;

  // CAMPO FECHA Y HORA DE CREACIÓN
  feCreacion: Date;

  // CAMPO USUARIO MODIFICADOR
  usActualizacion: string;

  // CAMPO TERMINAL DE MODIFICACIÓN
  ipActualizacion: string;

  // CAMPO FECHA Y HORA DE MODIFICACIÓN
  feActualizacion: Date;

  /**-------nuevo------ */

  //UBIGEO DE NACIMIENTO O DE ORIGEN
  descUbigeoNacimientoCompleto: string;
}

File: portafolio-backend/src/main/java/com/portafolio/portafoliobackend/dtos/TblPerfilDTO.java
package com.portafolio.portafoliobackend.dtos;

import java.util.Date;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class TblPerfilDTO {

    // CAMPO IDENTIFICADOR DE PERFIL
    private Long idPerfil;

    // CAMPO IDENTIFICADOR DE UBIGEO NACIMIENTO DEL PERFIL
    private Long idUbigeo;

    // CAMPO NUMERO DE DNI DEL PERFIL
    private Long nuDniPerfil;

    // CAMPO NOMBRE DE PERFIL
    private String noPerfil;

    // CAMPO APELLIDOS DEL PERFIL
    private String apPerfil;

    // CAMPO DIRECCIÓN DEL PERFIL
    private String dirPerfil;

    // CAMPO TELEFONO DEL PERFIL
    private Long telPerfil;

    // CAMPO CORREO ELECTRONICO DEL PERFIL
    private String emailPerfil;

    // CAMPO FECHA DE NACIMIENTO DEL PERFIL
    private Date feNacimientoPerfil;

    // NOMBRE ORIGINAL DE LA IMAGEN
    private String imgPerfil;

    // TIPO DE IMAGEN
    private String tipoImg;

    // NOMBRE DE LA IMAGEN
    private byte[] codImg;

    // DESCRIBIR SOBRE MÍ DE COMO SOY
    private String sobreMi;

    // RESUMEN DE LO QUE HAGO
    private String resumen;

    // CAMPO ESTADO DEL REGISTRO. LOS POSIBLES VALORES SON: "1" = ACTIVO Y "0" =
    // INACTIVO
    private String esRegistro;

    // CAMPO USUARIO CREADOR
    private String usCreacion;

    // CAMPO TERMINAL DE CREACIÓN
    private String ipCreacion;

    // CAMPO FECHA Y HORA DE CREACIÓN
    private Date feCreacion;

    // CAMPO USUARIO MODIFICADOR
    private String usActualizacion;

    // CAMPO TERMINAL DE MODIFICACIÓN
    private String ipActualizacion;

    // CAMPO FECHA Y HORA DE MODIFICACIÓN
    private Date feActualizacion;

    /**--------nuevo------- */
    // Ubigeo nacimiento u origen
    private String descUbigeoNacimientoCompleto;
}
